GULP

maclaren23 opened the pull request:
Создал сборку Gulp. Перенёс проект DLK в сборку Gulp.

maclaren23 committed:
GULP

Создал сборку Gulp. Перенёс проект DLK в сборку Gulp.

maclaren23 committed:
Добавил страницы (animation, svg)

Shookl commented on #src/catalog.html:
```diff
@@ -0,0 +1,93 @@
+              toys
+            </a>
+          </li>
+          <li class="categories__item">
```
а почему эти штуки не выносишь в компонеты?

maclaren23 replied:
ок, сделаю.

Shookl commented on #src/scss/common.scss:
```diff
@@ -0,0 +1,115 @@
+  &__item {
+    margin-right: 51px;
+    position: relative;
+    & a {
```
здесь & можно не писать. он нам нужен чтобы `&__list` такая запись стала `.breadcrumbs__list`. без амперсанда иначе она отобразиться `__list`. когда мы добавляем по такой системе цельные классы или теги, то мы просто дублируем еще раз родительский класс. не критично, но обычно в таких случаях амперсанд не пишут

Shookl commented on #src/animation.html:
```diff
@@ -0,0 +1,72 @@
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.min.css" />
+    <link rel="stylesheet" href="css/common.min.css" />
```
галп же и так минимизирует код, здесь нам нет нужны подключать такие файлы

maclaren23 replied:
у меня в папке build компилируется 2 файла css (простой и минимизированный), поэтому подключаю именно минимизированный css

maclaren23 committed:
all pages

1)Добавил страницу Grid.
2)Адаптив всех страниц.
3)Оптимизация изображений.
4)Замена яндекс карты на google.
5)Добавление аудио файла на главной странице.
6)Залил сайт на github page.
